nsv13/code/modules/power/stormdrive.dm

```diff
@@ -353,7 +359,15 @@ Takes  plasma and outputs superheated plasma and a shitload of radiation.
 		return
 	cut_overlays()
 	if(can_cool()) //If control rods aren't destroyed.
-		add_overlay("rods_[control_rod_state]")
+		switch(control_rod_percent)
+			if(0 to 24)
+				add_overlay("rods_1")
+			if(25 to 49)
+				add_overlay("rods_2")
+			if(50 to 74)
+				add_overlay("rods_3")
+			if(75 to 100)
+				add_overlay("rods_4")
```

Comment:
Bug: Setting the control rod percentage to a number in between one of these ranges (e.g. 24.5) results in the rods disappearing entirely, as if set to rods_4. I don't know byond code well enough to know the proper fix.

Suggestion: In this current distribution, rod position 1 and 2 use the same icon, rod 3 uses icon 2, and icon 3 never gets used except for mostly-pointless manual assignments.

I suggest the ranges should be 0-10 for "rods_1"/fully raised (because that is the range at which the reactor will actually explode), 11-24 for "rods_2"/mostly-raised (between ~450-200 degrees, i.e. what was considered at least "hot" in the old definitions), 25-52  for "rods_2"/mostly-inserted ("safe" temperatures, within the realm of rod position 3), and 53-100 for "rods_4"/SCRAM (This is the range at which the reaction will actually just die, slowly at about 53% and pretty rapidly once you go above 55%.

That way, the visual positions of the control rods will provide hints to engineers who didn't do the math of the expected results of their terrible decisions.

Reply:
Not so much of a bug as a lazy get around while I did other things.

Reply:
Fixed.